tests/utils.py

```diff
@@ -19,42 +19,25 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-import asyncio
-from abc import ABC, abstractmethod
+import os
 
 
-class ButlerIngester(ABC):
-    """Interface class for processing files for ingestion into a butler.
+def removeEntries(directory):
+    """Remove given directory and all entries below it
```

Comment:
Blank line separating the summary from the parameter description is missing.

Reply:
I removed this file and replaced it to calls to shutil.rmtree